app/controllers/IdentityController.scala

```diff
@@ -9,15 +9,16 @@ import monitoring.SafeLogger._
 import play.api.mvc._
 import play.api.libs.circe.Circe
 import services.IdentityService
-import codecs.CirceDecoders._
 import cats.implicits._
+import models.identity.CookiesResponse
 
 import scala.concurrent.ExecutionContext
 
 class IdentityController(
     identityService: IdentityService,
     components: ControllerComponents,
-    actionRefiners: CustomActionBuilders
+    actionRefiners: CustomActionBuilders,
+    guardianDomain: String
```

Comment:
general comment I thought I'd throw in for food for thought:
in new code I try not to pass String around on its own where possible as the type checker won't help us much.  Usually I make a `case class GuardianDomain(value: String) extends AnyVal` and then take parameter `guardianDomain: GuardianDomain`
This is indeed a more high level type form of programming, so this is more of a facet of the static vs dynamic types debate.